Sofa/Component/Mass/src/sofa/component/mass/MeshMatrixMass.h

```diff
@@ -149,7 +149,8 @@ class MeshMatrixMass : public core::behavior::Mass<DataTypes>
         m_massTopologyType = t;
     }
 
-    int getMassCount() {
+    auto getMassCount()
```

Comment:
I'm not in favor of using auto in the return type of function because it makes function un-trustable as a way to detect and track internal changes propagation to the public api.